mayastor-test/test_csi.js

```diff
@@ -24,9 +24,11 @@ const protoLoader = require('@grpc/proto-loader');
 // possible only with grpc-uds.
 const grpc = require('grpc-uds');
 const common = require('./test_common');
+const mayastorProto = require('./mayastor_proto');
```

Comment:
nit; you could just do a require('./mayastor_proto).getConstants() here and get rid of line 31

Reply:
Fixe, https://github.com/openebs/Mayastor/pull/125/commits/85459d5655e148ae4eab771a3685a17e3d2089ab